game/state/city/vequipment.cpp

```diff
@@ -138,8 +139,27 @@ void VEquipment::update(int ticks)
 	}
 }
 
+void VEquipment::noAmmoToReload(GameState &state, VEquipment *equipment) const {
```

Comment:
I believe both "gamestate" and "equipment" arguments can be const here?

(I know it's not consistently applied through the codebase right now, but I prefer "const if possible")